dom/webidl/MozVisibleSearchEnginesChangeEvent.webidl

```diff
@@ -0,0 +1,10 @@
+[Constructor(DOMString type, VisibleSearchEnginesChangeEventInit eventInitDict)]
+interface MozVisibleSearchEnginesChangeEvent : Event {
```

Comment:
I think the name is overly specific (i.e., the "visible" bit) - if the engine is visible or not is something either the engine knows, or can somehow be queried. Maybe we can just call it `MozSearchEngineChangeEvent`.